packages/sources/css/src/components/structure/list/src/index.css

```diff
@@ -18,6 +18,21 @@
   padding: 0;
 }
 
+.vtmn-list a.vtmn-list__link {
+  display: flex;
+  inline-size: 100%;
+  text-decoration: inherit;
+  color: inherit;
+  padding-right: 2.5rem; /* to avoid the padding-inline from child */
+}
+
+.vtmn-list li > a.vtmn-list__link[aria-disabled='true'] {
+  color: currentcolor;
+  display: inline-block; /* For IE11/ MS Edge bug */
+  pointer-events: none;
+  text-decoration: none;
+}
+
```

Comment:
Today the button exist. It is the component without the href and you can access on the event with the on:click.

With this attribute, we want to add the redirection. So only the `<a>` will be affected by this css.

So yes and no, that because:
- the `.vtmn-list__link[aria-disabled='true']` is specific to the node `<a>`
- We can changes tomorrow with something else but it should be always the same behaviour on the top level (`<a>`). 

So keep the `<a>` selector for me is a secure option that said, if you put something else that a `<a>` on top, it will doesn't work.

Reply:
Not sure about the `It is the component without the href and you can access on the event with the on:click`. Should we not use an accessible and semantic button as we are doing for the `<a>` ?